golang/src/user-sessions/main_test.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import "testing"
+
+func TestConvertBlankSession(t *testing.T) {
+	record := &UserSessionRecord{
+		ID:      "test_id",
+		Session: "",
+		UserID:  "test_user_id",
+	}
+	actual, err := convert(record, false)
```

Comment:
Unlikely to go wrong but still might be worth adding a quick test or several which call `convert(..., true)`, since it looks like all of these pass `false`.